redacted-compiler-plugin/src/main/kotlin/dev/zacsweers/redacted/compiler/RedactedIrVisitor.kt

```diff
@@ -161,6 +174,10 @@ internal class RedactedIrVisitor(
     return hasAnnotation(redactedAnnotation)
   }
 
+  private fun IrProperty.isUnredacted(): Boolean {
```

Comment:
nit: let's make it an extension property instead

Reply:
Want me to update the `isRedacted()` extension method as well?

Reply:
Sounds good